pulp_rpm/app/serializers/advisory.py

```diff
@@ -231,6 +232,16 @@ def validate(self, data):
         validated_data = super().validate(update_record_data)
         return validated_data
 
+    # pulpcore 3.22 feature
+    # TODO upload has to be able to "find" a unique advisory, and id is Not It?
+    # when uploading content second time I don't get error but already existing content
+    def retrieve(self, validated_data):
+        content = UpdateRecord.objects.filter(
+            id=validated_data["id"],
+            pulp_domain=get_domain(),
```

Comment:
Unless you need to use the Domain's name, I would just use `get_domain_pk()` for these filters.